manager/controllers/common_test.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"app/base/core"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+// Content type variables
+var contentTypeCSV = "text/csv"
+var contentTypeJSON = "application/json"
+
+// Init request
+func PrepareRequest(method string, url string, body io.Reader) (w *httptest.ResponseRecorder, req *http.Request) {
```

Comment:
doesn't have to be exported 
```suggestion
func prepareRequest(method, url string, body io.Reader) (w *httptest.ResponseRecorder, req *http.Request) {
```

Reply:
I agree.